e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -99,14 +98,14 @@ describe("viem send tx", () => {
 	}).timeout(10_000);
 
 	it("send fee currency tx with explicit gas fields and check receipt", async () => {
-		const [maxFeePerGas, tip] = await getGasFees(publicClient, 2n, process.env.FEE_CURRENCY);
+		const baseFeeForFeeCurrency = await getBaseFee(publicClient, process.env.FEE_CURRENCY);
 		const request = await walletClient.prepareTransactionRequest({
 			to: "0x00000000000000000000000000000000DeaDBeef",
 			value: 2,
 			gas: 171000,
 			feeCurrency: process.env.FEE_CURRENCY,
-			maxFeePerGas: maxFeePerGas,
-			maxPriorityFeePerGas: tip,
+			maxFeePerGas: baseFeeForFeeCurrency*10n,
+			maxPriorityFeePerGas: 1n,
```

Comment:
You're still getting a failure here in CI.
This is because the node has a min-tip of 1 configured - however since this is denominated in fee-currency
you also have to convert the min-tip into the fee-currency.
In this case this is 2 - the log message is misleading because the unconverted value is logged as the expected value. I pushed a fix for this to this branch.

Reply:
Thanks, so I need to set a tip here of 2 or more?

Reply:
Yes, I didn't look up the rate, but the next CI run will reveal this. I got 2 locally so I assume the rate is `1 : 2`

Reply:
Ok instead I'll just convert to the fee currency, it's safer.